djimporter/templates/djimporter/importlog_detail.html

```diff
@@ -58,8 +58,17 @@ <h3>Errors for <strong>{{ object.input_file }}</strong></h3>
               <tr>
                 <td class="align-middle">{{ forloop.counter }}</td>
                 <td class="align-middle">{{ error.line }}</td>
-                <td class="align-middle">{{ error.field }}</td>
-                <td>{{ error.message }}</td>
+                {% if error.message %}
+                  <td class="align-middle">{{ error.field }}</td>
+                  <td>{{ error.message }}</td>
+                {% else %}
```

Comment:
Maybe we can add a comment to refer that this code is to support legacy error format (prior to v0.2)?